dev/packages/beats/zeek-0.0.1/dataset/capture_loss/agent/stream/stream.yml

```diff
@@ -0,0 +1,16 @@
+input: log
+paths:
+{{#each paths}}}}
+  - {{this}}
+{{/each}}
+exclude_files: [".gz$"]
+tags: {{this.tags}}
```

Comment:
Should this be just `tags`?

Reply:
"this" has been added as a workaround for "range":

```
{{#each animals}}}}
  - {{this.species}}
     {{this.size}}
     {{this.name}}
{{/each}}
```

It should cover all placeholders for variable, not analyzing if the used variable is inside the loop or not.